config/fund_loader_config/digital_planning/dpi_r2.py

```diff
@@ -0,0 +1,149 @@
+from datetime import datetime
+from datetime import timezone
+
+from config.fund_loader_config.common_fund_config.fund_base_tree_paths import (
+    DPI_R2_BASE_PATH,
+)
+from config.fund_loader_config.logo import DLUHC_LOGO_PNG
+
+DPI_FUND_ID = "f493d512-5eb4-11ee-8c99-0242ac120002"
+DPI_ROUND_2_ID = "0059aad4-5eb5-11ee-8c99-0242ac120002"
+APPLICATION_BASE_PATH = ".".join([str(DPI_R2_BASE_PATH), str(1)])
+ASSESSMENT_BASE_PATH = ".".join([str(DPI_R2_BASE_PATH), str(2)])
+DPI_R2_OPENS_DATE = datetime(
+    2023, 10, 19, 10, 0, 0, tzinfo=timezone.utc
+)  # 2023-10-17 10:00:00
+DPI_R2_DEADLINE_DATE = datetime(
+    2023, 12, 1, 11, 59, 0, tzinfo=timezone.utc
+)  # 2023-11-1 11:59:00
+DPI_R2_ASSESSMENT_DEADLINE_DATE = datetime(
+    2024, 1, 31, 12, 0, 0, tzinfo=timezone.utc
+)  # 2023-01-31 12:00:00
+
```

Comment:
@adamdavies1 Can you add a variable for privacy notice so we can add the url in easily once we get the URL from the fund team?

Just like you have done in Line 23